### PR TITLE
create edit-task feature

### LINKED ISSUE #9: Add 'Edit Task' function

### PR DESCRIPTION
**Pull request by lakshay-angrish**
ss
Closes #9 
We can now edit a task, as well as delete it.

### CODE REVIEW

**Comment by lakshay-angrish** [comment]
@wayne9499 I was not able to display the current priority in the edit task dialog. You should look into that.

**Comment by wayne9499** [comment]
please add screenshot of edit-task dialog box

File: server.js [hunk 1]
```diff
@@ -96,6 +95,36 @@ app.post('/newTask', (req, res) => {
   });
 });
 
+app.post('/editTask', (req, res) => {
+  Task.replaceOne({
+    _id: ObjectID(req.body._id)
+  },
+  req.body
```
Comment (wayne9499) [inline server.js:102]: whats the use of this 'req.body'. you should remove this line

**Comment by lakshay-angrish** [comment]
I deleted your comment by mistake.
The first argument to `replaceOne()` is the filter, the second is what to replace the document with, which is `req.body` in this case.

**Comment by lakshay-angrish** [comment]
> please add screenshot of edit-task dialog box

done

**Comment by wayne9499** [comment]
> I deleted your comment by mistake.
> The first argument to `replaceOne()` is the filter, the second is what to replace the document with, which is `req.body` in this case.

yes, btw i deleted when i understand

**Pull request merged**